src/Repository/index.js

```diff
@@ -61,17 +61,17 @@ class Repository {
         return multipleFilesByPath
     }
 
-    async getHeadRef() {
+    async getHeadRef(defaultBranch) {
         const result = await this.github.git.getRef({
             owner: this.owner,
             repo: this.repo,
-            ref: `heads/master`,
+            ref: `heads/${defaultBranch}`,
         })
         return result.data.object.sha
     }
 
-    async createBranch(branchName) {
-        const fromSha = await this.getHeadRef()
+    async createBranch(branchName, defaultBranch) {
```

Comment:
:tada:

Reply:
[minor] usually switch parameters to named, if more than 1 parameter to the function